.pre-commit-config.yaml

```diff
@@ -1,4 +1,6 @@
 ---
+ci:
```

Comment:
@kaxil why we did these changes? just curious :)

Reply:
Since we are already running mypy checks separately in CI -- separate from pre-commit hook